packages/flutter/lib/src/widgets/form.dart

```diff
@@ -291,18 +291,44 @@ class FormState extends State<Form> {
   /// returns true if there are no errors.
   ///
   /// The form will rebuild to report the results.
+  ///
+  /// See also:
+  ///  * [validateGranularly], which returns a [Map] describing the validation
+  /// status for each [FormField]
   bool validate() {
     _hasInteractedByUser = true;
     _forceRebuild();
     return _validate();
   }
 
-  bool _validate() {
+
+  /// Validates every [FormField]s that is a descendant of this [Form], and
+  /// returns a [List] of [FormFieldState] of the invalid field(s) only, if any.
+  ///
+  /// To get the absolute validation value of the Form, Consider using [validate]
+  ///
+  /// Common usage of this method is when you need draw the user's attention
```

Comment:
Maybe: This method can be useful when ... / can be used to highlight fields with errors.